web/src/lib/i18n/locales/en-EN.json

```diff
@@ -0,0 +1,763 @@
+{
+    "thirty_minutes": "30 minutes",
+    "one_hour": "1 hour",
+    "six_hours": "6 hours",
+    "one_day": "1 day",
+    "seven_days": "7 days",
+    "thirty_days": "30 days",
+    "three_months": "3 months",
+    "one_year": "1 year",
```

Comment:
For fields like this, should we use plurals? Per https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format (though I may have made a mistake here):
```
{"hour": "{count, plural, one {count day} other {{count, number} days}}
```
And then using that like
```
{$_('hour', { values: { count: 3 }})}
```

Reply:
Plurals are very welcomed, especially for non-germanic languages (etc. slavic).